src/btree/bt_vrfy.c

```diff
@@ -582,6 +583,105 @@ __verify_addr_ts(WT_SESSION_IMPL *session, WT_REF *ref, WT_CELL_UNPACK *unpack,
     return (0);
 }
 
+/*
+ * __verify_btree_id_with_meta --
+ *     With a given btree id, iterate through the metafile table and check if the btree id exists in
+ *     any table. If exists, we place the table URI into a variable otherwise should return
+ *     WT_NOTFOUND.
+ */
+static int
+__verify_btree_id_with_meta(WT_SESSION_IMPL *session, uint32_t btree_id, const char **uri)
+{
+    WT_CONFIG_ITEM id;
+    WT_CURSOR *meta_cursor;
+    WT_DECL_RET;
+    const char *meta_value;
+
+    WT_ERR(__wt_metadata_cursor(session, &meta_cursor));
+    while ((ret = meta_cursor->next(meta_cursor)) == 0) {
+        WT_ERR(meta_cursor->get_value(meta_cursor, &meta_value));
+        if ((ret = __wt_config_getones(session, meta_value, "id", &id)) == 0) {
+            if (btree_id == __wt_strtouq(id.str, NULL, 10)) {
+                meta_cursor->get_key(meta_cursor, uri);
+                break;
+            }
+        }
+        WT_ERR_NOTFOUND_OK(ret);
+    }
```

Comment:
I'm confused why we're calling `__wt_strtouq()`.
Doesn't `__wt_config_getones()` return the `int64_t` value of `id` in `id.val`?